core/src/main/java/com/linecorp/armeria/client/Clients.java

```diff
@@ -59,23 +62,23 @@
      * @param options the {@link ClientOptions}
      *
      * @throws IllegalArgumentException if the scheme of the specified {@code uri} or
-     *                                     the specified {@code clientType} is unsupported for the scheme
+     *                                  the specified {@code clientType} is unsupported for the scheme
      */
     public static <T> T newClient(String uri, Class<T> clientType, ClientOptions options) {
         return newClient(ClientFactory.DEFAULT, uri, clientType, options);
     }
 
     /**
-     * Creates a new client that connects to the specified {@code uri} using an alternative
+     * Creates a new client that connects to the specified {@code uri} using the specified
      * {@link ClientFactory}.
      *
-     * @param factory an alternative {@link ClientFactory}
+     * @param factory the specified {@link ClientFactory}
```

Comment:
I'm sorry but I think `an alternative {@link ClientFactory}` would be better from here. I wanted to rewrite the method description only. :-)

Reply:
Oh I didn't apply a change...!
Sorry, let me apply it!